README.md

```diff
@@ -12,9 +12,10 @@ I use flex-box, img and image-background and grid to create this.
 
```

Comment:
- [ ] Please add the `installations and setup` section and project repo `issues page link` and `license` to your Readme file, because it makes your Readme more informative.